From f6429960b48dab58643af33f70f0bc56aac1f5a7 Mon Sep 17 00:00:00 2001
From: "Arsalan H. Awan" <Arsalan_Awan@mentor.com>
Date: Mon, 17 Sep 2018 19:43:49 +0500
Subject: [PATCH 1/2] vktrace: append cxx flags to support cross compilation

To support cross compilation, we need to set the variable
CMAKE_CXX_FLAGS_LIST such that the CXX flags set by user
for cross compilation are also included here.

Signed-off-by: Arsalan H. Awan <Arsalan_Awan@mentor.com>
---
 vktrace/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vktrace/CMakeLists.txt b/vktrace/CMakeLists.txt
index 68bb642d8..00df0d1eb 100644
--- a/vktrace/CMakeLists.txt
+++ b/vktrace/CMakeLists.txt
@@ -64,7 +64,7 @@ if(MSVC)
     set(CMAKE_CXX_FLAGS_RELEASE_LIST "/O2 ${CMAKE_CXX_FLAGS_RELEASE_LIST} /DNDEBUG")
     set(CMAKE_CXX_FLAGS_DEBUG_LIST "/Od /D_DEBUG")
 else()
-    set(CMAKE_CXX_FLAGS_LIST "-Wall -Wextra")
+    set(CMAKE_CXX_FLAGS_LIST "${CMAKE_CXX_FLAGS} -Wall -Wextra")
     set(CMAKE_CXX_FLAGS_RELEASE_LIST "-O2 ${CMAKE_CXX_FLAGS_RELEASE_LIST} -DNDEBUG")
     set(CMAKE_CXX_FLAGS_DEBUG_LIST "-g -O0 -D_DEBUG")
 endif()
-- 
2.11.1

